core/src/test/resources/META-INF.services/org.axonframework.messaging.annotation.ParameterResolverFactory

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Isn't this file a duplicate?

Reply:
This one is for tests :)

Reply:
Baah of course, thanks for pointing that out.

Reply:
Actually, the runtime version of this registration should be enough.